### PR TITLE
Update tls-1.0-and-1.1-deprecation-for-office-365.md

### PR DESCRIPTION
**Pull request by msbemba**
Added references to provide guidance to help make sure that clients are using TLS 1.2 or a later version and to disable TLS 1.0 and 1.1.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4177

File: microsoft-365/compliance/tls-1.0-and-1.1-deprecation-for-office-365.md [hunk 1]
```diff
@@ -86,3 +86,19 @@ You'll need to update applications that call Microsoft 365 APIs over TLS 1.0 or
 ## More information
 
 For more information, see [Preparing for the mandatory use of TLS 1.2 in Office 365](https://support.microsoft.com/help/4057306/preparing-for-tls-1-2-in-office-365).
+
+## References
+
+The following resources provide guidance to help make sure that your clients are using TLS 1.2 or a later version and to disable TLS 1.0 and 1.1.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/tls-1.0-and-1.1-deprecation-for-office-365.md]:
```suggestion
The following resources provide guidance to help make sure that your clients are using TLS 1.2 or a later version and to disable TLS 1.0 and 1.1:
```

File: microsoft-365/compliance/tls-1.0-and-1.1-deprecation-for-office-365.md [hunk 2]
```diff
@@ -86,3 +86,19 @@ You'll need to update applications that call Microsoft 365 APIs over TLS 1.0 or
...
+
+- For Windows 7 clients that connect to Office 365, make sure that TLS 1.2 is the default secure protocol in WinHTTP in Windows. For more information see [KB 3140245 - Update to enable TLS 1.1 and TLS 1.2 as a default secure protocols in WinHTTP in Windows](https://support.microsoft.com/help/3140245/update-to-enable-tls-1-1-and-tls-1-2-as-a-default-secure-protocols-in).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/tls-1.0-and-1.1-deprecation-for-office-365.md]:
```suggestion
- For Windows 7 clients that connect to Office 365, make sure that TLS 1.2 is the default secure protocol in WinHTTP in Windows. For more information, see [KB 3140245 - Update to enable TLS 1.1 and TLS 1.2 as default secure protocols in WinHTTP in Windows](https://support.microsoft.com/help/3140245/update-to-enable-tls-1-1-and-tls-1-2-as-a-default-secure-protocols-in).
```

File: microsoft-365/compliance/tls-1.0-and-1.1-deprecation-for-office-365.md [hunk 3]
```diff
@@ -86,3 +86,19 @@ You'll need to update applications that call Microsoft 365 APIs over TLS 1.0 or
...
+- To start addressing weak TLS use by removing TLS 1.0 and 1.1 dependencies, see [TLS 1.2 support at Microsoft](https://cloudblogs.microsoft.com/microsoftsecure/2017/06/20/tls-1-2-support-at-microsoft/).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/tls-1.0-and-1.1-deprecation-for-office-365.md]:
```suggestion
- To address weak TLS usage by removing TLS 1.0 and 1.1 dependencies, see [TLS 1.2 support at Microsoft](https://cloudblogs.microsoft.com/microsoftsecure/2017/06/20/tls-1-2-support-at-microsoft/).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@WorkShay Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@WorkShay Gentle reminder please merge this PR. Thanks.

**Comment by yogkumgit** [comment]
@Kateyanne please let us know who can merge it. Thanks

**Comment by Kateyanne** [comment]
@JohanFreelancer9 would you please check again that the url is correct given our new instructions?

**Comment by msbemba** [comment]
@Kateyanne This document was updated before the instructions were updated. I have updated the URL ( Only one belongs to docs. microsoft.com) and it should be good to merge

**Pull request merged**